src/main/java/org/prebid/server/handler/SetuidHandler.java

```diff
@@ -104,12 +105,22 @@ public SetuidHandler(long defaultTimeout,
         this.metrics = Objects.requireNonNull(metrics);
         this.timeoutFactory = Objects.requireNonNull(timeoutFactory);
 
-        cookieNameToSyncType = bidderCatalog.names().stream()
+        cookieNameToSyncType = collectMap(bidderCatalog);
```

Comment:
While you are here, you can also place `this` here.

Reply:
Got it.